### ISSUE TITLE
Text Disappears from end the text message

### ISSUE THREAD

**Issue by nacr** [emilsedgh/modem#34]
Hello @emilsedgh first of all nice work.

I have been testing this package and I have found one problem.
Text is disappearing from the end of the message.

In my tests I have detected that 6 characters disappear. Can You Please check this?

**Comment by emilsedgh** [comment]
Hi Nuno.

Unfortunately this package is not really maintained anymore as I don't have a device and motivation  to test it.

However, I think this bug has been reported before. And patches and fixes are around.

I think the patches could be found in the pdu.js package.

**Comment by derit** [comment]
yes i have same problem, last length of text from message, is not send

**Comment by derit** [comment]
i send  **hello world** 
and reiceve   **hello worl**

**Comment by mhabib555** [comment]
I was facing the same issue. As this package is not being update so i tried https://github.com/alexislg2/modem 
and its working great

**Comment by nacr** [comment]
@mhabib555 tks and sorry for the late message.
